pom.xml

```diff
@@ -105,6 +105,18 @@
       <artifactId>xml-resolver</artifactId>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <!--scope>test</scope-->
```

Comment:
Hmmm, I don't remember why I commented out the scope. It might be because xerces didn't have a src/test/java directory until I added one. Yeah, maybe I originally put my tests in their tests directory. But I just built xerces and ran my tests with <scope>test</scope> and it went OK. I'll change that.